src/vs/editor/contrib/links/links.ts

```diff
@@ -299,10 +299,12 @@ class LinkDetector implements editorCommon.IEditorContribution {
 			return this.openerService.open(uri, { openToSide });
 
 		}, err => {
+			const errorMessage = err ? err.message : undefined;
+			const messageOrError = errorMessage || err;
 			// different error cases
```

Comment:
It works as expected but we can probably improve the readability by checking the types of `err` instead of relying `any`. You can use the logic in https://github.com/microsoft/vscode/blob/734c85e96db19ccf7e6c9b5a28bcc276a900351a/extensions/css-language-features/server/src/utils/runner.ts#L8-L16 to generate the `messageOrError`.